### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by afzaal-ahmad-zeeshan**
Added an await expression for usage.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -155,6 +155,9 @@ All the calls have async variants:
 
 ```csharp
 var response = client.IndexAsync(tweet, idx => idx.Index("mytweetindex")); // returns a Task<IndexResponse>
+
+// Or, in an async-context
+var response = await client.IndexAsync(tweet, idx => idx.Index("mytweetindex")); // unwraps a Task<IndexResponse>
```
Comment (Mpdreamz) [inline readme.md]: not sure I like the `unwrap` here.  That term is also used in TPL to go from `Task<Task<T>>` to just `Task<T>`. 
maybe change `unwrap` with `await`.

```suggestion
var response = await client.IndexAsync(tweet, idx => idx.Index("mytweetindex")); // awaits a Task<IndexResponse>
```

**Comment by Mpdreamz** [comment]
Thanks for the PR @afzaal-ahmad-zeeshan, I've made one suggested edit.

File: readme.md (hunk 1 above)
Comment (afzaal-ahmad-zeeshan) [reply readme.md]: Thanks, yep, you are right, `await` sounds suitable here.

**Comment by Mpdreamz** [comment]
Thanks @afzaal-ahmad-zeeshan for updating the PR with my nitpick 😄

**Pull request merged**